src/gui/tray/activitylistmodel.cpp

```diff
@@ -394,6 +360,24 @@ void ActivityListModel::activitiesReceived(const QJsonDocument &json, int status
     }
 
     _activityLists.append(list);
+}
+
+void ActivityListModel::activitiesReceived(const QJsonDocument &json, int statusCode)
+{
+    const auto activities = json.object().value("ocs").toObject().value("data").toArray();
```

Comment:
`QStringLiteral("ocs")` and `QStringLiteral("data")`

Reply:
Now use QStringLiterals